cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/DefaultStore.java

```diff
@@ -781,10 +781,9 @@ private boolean scanApplicationsWithReorder(ScanApplicationsRequest request,
   }
 
   @Override
-  public Map<ApplicationId, ApplicationSpecification> getApplications(Collection<ApplicationId> ids) {
+  public Map<ApplicationId, ApplicationMeta> getApplications(Collection<ApplicationId> ids) {
```

Comment:
This store interface is currently only used by `GET /appdetail`, will be used by SCM pushApps flow in coming PRs